nwc_backend/__init__.py

```diff
@@ -75,7 +76,7 @@ async def serve_frontend(path: str) -> Response:
             return Response("No frontend build path provided", status=500)
         static_folder: str = app.static_folder
         path = path.replace(base_path.strip("/"), "")
-        if path != "" and os.path.exists(static_folder + "/" + path):
+        if path != "" and os.path.exists(safe_join(static_folder, path)):
             return await send_from_directory(static_folder, path)
```

Comment:
Ah it looks like send_from_directory is already using safe_join: https://github.com/pallets/quart/blob/b5593ca4c8c657564cdf2d35c9f0298fce63636b/src/quart/helpers.py#L28,sso we can drop this PR.